Recommend to check `symbolicator` if source maps are not working on Self-Hosted (#11765)
getsentry/sentry-docs: platform-includes/sourcemaps/legacy-troubleshooting/javascript.mdx

If you're <PlatformLink to="/sourcemaps/uploading/hosting-publicly/">hosting your source maps publicly</PlatformLink>, make sure you have the "Enable JavaScript source fetching" option activated under **[Settings] > Projects > Select your project > General Settings**.

> ## (Self-Hosted Sentry) Verify the `symbolicator` service is operating normally
> 
> If you are running a self-hosted version of Sentry, verify that the `symbolicator` service/container is operating normally.
> You can do so by checking the container's logs.

Reviewer: ```suggestion
## (Self-Hosted Sentry) Verify the `symbolicator` service is operating normally

If you're running a self-hosted version of Sentry, you can verify that the `symbolicator` service/container is operating normally by checking the container's logs.
```
